style.css

```diff
@@ -0,0 +1,588 @@
+  top: 19px;
+  left: 11px;
+  height: 1px;
+  width: 18px;
+  border-radius: 2px;
+  cursor: pointer;
+}
+
+.menu-button::before {
+  content: '';
+  top: -6px;
+  left: 0;
+}
+
+.menu-button::after {
+  content: '';
+  top: 6px;
+  left: 0px;
+}
+
+.logoCont {
+  grid-area: 1 / 2 / 2 / 3;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+}
+
+.logoCont > a {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+}
+
+#logoImg {
+  position: absolute;
+  left: 10px;
+  width: 95px;
+  top: 16px;
+}
+
+.searchbarCont {
+  grid-area: 1 / 3 / 2 / 4;
+  display: flex;
+  flex-wrap: nowrap;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+
+.searchbarCont > input {
+  display: inline-block;
+  border-style: none;
+  height: 36px;
+  width: 55%;
+  border-radius: 25px 0 0 25px;
+  border: 1px solid rgb(204, 201, 201);
+  box-shadow: inset 0px 1px 2px rgba(0, 0, 0, 0.081);
+  font-size: 1rem;
+  padding-left: 1rem;
+  z-index: 1;
+}
+
+.searchbarCont > input:focus {
+  outline: 1px solid blue;
+}
+
+.searchbarCont > input::placeholder {
+  font-size: 1rem;
+  color: rgb(144, 144, 144);
+}
+
+.srchIconCont {
+  display: inline-block;
+  height: 38px;
+  width: 60px;
+  border-radius: 0 25px 25px 0;
+  background-color: rgb(248, 248, 248);
+  border: 1px solid rgb(204, 201, 201);
+  border-left: 0ch;
+  cursor: pointer;
+}
+.srchIconCont:hover {
+  background-color: gainsboro;
+}
+
+#magGlass {
+  position: relative;
+  top: 9px;
+  left: 20px;
+  height: 20px;
+  cursor: pointer;
+  z-index: 1;
+}
+
+.micCont {
+  display: inline-block;
+  background-color: rgb(248, 248, 248);
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  margin-inline-start: 9px;
+  cursor: pointer;
+}
+
+.micCont:hover {
+  background-color: gainsboro;
+}
+
+#mic {
+  position: relative;
+  top: 10px;
+  left: 13px;
+  height: 21px;
+}
+
+.buttonsCont {
+  position: absolute;
+  grid-area: 1 / 4 / 2 / 5;
+  align-items: center;
+  justify-content: center;
+}
+
+.crtIconCont {
+  position: relative;
+  top: 8px;
+  left: 21px;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.crtIconCont:hover {
+  background-color: gainsboro;
+}
+
+#crtImg {
+  position: relative;
+  width: 21px;
+  height: 18px;
+  top: 10px;
+  left: 9px;
+}
+
+.bellIconCont {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  top: -32px;
+  left: 69px;
+  cursor: pointer;
+}
+
+.bellIconCont:hover {
+  background-color: gainsboro;
+}
+
+#bellIcon {
+  width: 18px;
+  height: 21px;
+  position: relative;
+  top: 9px;
+  left: 11px;
+}
+
+.prflImgCont {
+  position: relative;
+  display: inline-block;
+  background-color: aqua;
+  border-radius: 50%;
+  width: 33px;
+  height: 33px;
+  top: -70px;
+  left: 132px;
+  font-size: 1.11rem;
+  cursor: pointer;
+}
+
+#profilePic {
+  position: relative;
+  top: -12px;
+  left: 3px;
+}
+
+.bodyCont {
+  display: flex;
```

Comment:
May try using `display: grid` for the body section so it can be easily changed with media queries for the mobile version.